src/lib/codeSamples/buildRequest.ts

```
@@ -25,19 +25,24 @@ function getPath(variables: Record<string, string>, pathParameters: OpenAPIV3.Pa
+  headerParameters: OpenAPIV3.ParameterObject[],
+  authorizations: PlaygroundSecurityScheme | PlaygroundSecurityScheme[] | null,
+) {
+  const resolvedHeaders = new Headers(headers)
 
   processParameters(variables, headerParameters, (key: string, value: string) => {
-    headers.set(key, value)
+    resolvedHeaders.set(key, value)
   })
 
   if (authorizations && !Array.isArray(authorizations)) {
     authorizations = [authorizations]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)